src/Aurora.Application/Entities/SearchRequest.cs

```diff
@@ -11,11 +11,11 @@ public class SearchRequest
         public Guid Id { get; set; }
 
         [MaxLength(255)]
-        public string SearchTerm { get; set; }
```

Comment:
bad practice to put nulls right here, make it at least as a nuallableldfsjgsd type